apps/meteor/app/livechat/server/sendMessageBySMS.js

```diff
@@ -66,9 +66,8 @@ callbacks.add(
 			extraData = Object.assign({}, extraData, { contentVariables });
 		}
 
-		if (message.customFields && message.customFields.messagingServiceSid) {
-			const { messagingServiceSid } = message.customFields;
-			extraData = Object.assign({}, extraData, { messagingServiceSid });
+		if (room.sms.messaging_service_sid) {
```

Comment:
Does this add messaging_service_sid to all of our bot responses to twilio, and not just the responses that include contentSid?  Is that the way we want to do it?  Are we sure that won't cause any confusion on the twilio side?  Or should we just be adding this when there is also the customFields with contentSid?